src/components/Messages/MessageList/index.js

```diff
@@ -111,6 +113,21 @@ export class MessageList extends MemoizedComponent {
 			const previousMessage = messages[i - 1];
 			const message = messages[i];
 			const nextMessage = messages[i + 1];
+			const { incomingCallAlert } = store.state;
+			const { ongoingCall } = store.state;
+
+			if (message.msg === 'Join my room to start the video call' && ongoingCall) {
```

Comment:
This logic won't work for other translations apart from English. We need to fix it